cookbook/Berksfile

```diff
@@ -1,3 +1,7 @@
 source 'https://supermarket.getchef.com'
 
 metadata
+
+cookbook 'mesos',
+         github: 'everpeace/cookbook-mesos',
+         ref: '85d4a8f5c441402844775d56b165919c99f8a546'
```

Comment:
this appears to conflict with `mesos` in [metadata.rb](https://github.com/evertrue/Singularity/blob/evertrue/eherot/use_mesos_cookbook/cookbook/metadata.rb#L15) for me